components/search/SearchResultsList.vue

```diff
@@ -5,7 +5,7 @@
     <b-list-group-item
       v-for="result in results"
       :key="result.europeanaId"
-      :to="result.linkTo"
+      :to="localePath({ name: 'record-all', params: { pathMatch: result.europeanaId.replace(/^\/+/g, '') } })"
```

Comment:
The `g` modifier seems redundant here given the `^` anchor.

To save using a regex, we could just do: 
```js
result.europeanaId.slice(1)
```